### ISSUE TITLE
Add accessible labels to buttons on Officer Profile screen

### ISSUE THREAD

**Issue by CR96** [WildAid/o-fish-android#415]
**Is your feature request related to a problem? Please describe.**
Buttons should have descriptive labels for Android TalkBack users.

**Describe the solution you'd like**
Add labels to Close and Change Profile Picture buttons.

**Comment by Sheeri** [comment]
Hello @CR96 , thanks for offering to work on this! I have assigned it to you.

**Comment by Sheeri** [comment]
Should the toggles also have labels? Right now the screen reader says "on switch" if the toggle is set to off, and 'off switch' if the toggle is set to on - but because there are 2 toggles ('at sea/not at sea' and 'dark mode') I worry that it's not clear what the user is turning on and off.

e.g. should we have: 'On switch for at sea', 'Off switch for not at sea', 'on switch for dark mode', 'off switch for dark mode'?

Please proceed with work on this, your other work was fantastic! Will merge soon.

**Comment by Sheeri** [comment]
(oh, @CR96 I'm sorry for assuming you wanted to work on that - I have unassigned)

**Comment by CR96** [comment]
@Sheeri No you're good! I was planning to open a PR for this

**Comment by Sheeri** [comment]
Great, @CR96 you've been assigned this one.

**Comment by CR96** [comment]
> Should the toggles also have labels? Right now the screen reader says "on switch" if the toggle is set to off, and 'off switch' if the toggle is set to on

The screen reader announces items as "State: Component," so it's saying "Off: Switch" as in "switch is off." TalkBack on Android 11 announces them as expected in my case.

> but because there are 2 toggles ('at sea/not at sea' and 'dark mode') I worry that it's not clear what the user is turning on and off.

What I can do is add the `android:labelFor` property to the toggle switches. This will prompt TalkBack to read the labels when a switch is in focus, e.g. "Off: Switch for Dark Mode."

**Comment by Sheeri** [comment]
Yes, that would be perfect - exactly what I was thinking, something so that there's a distinction between the two toggle switches.
